doc/source/configuration/block-storage/drivers/lightbits-lightos-driver.rst

```diff
@@ -110,6 +114,12 @@ Then create a new volume with one of these volume types:
 
    $ openstack volume create --size <size> --type <type name> <vol name>
 
+Create a volume type for multi tenancy:
+
+.. code-block:: console
+
+   $ openstack volume type create --property lightos:project_name=project-01 type-with-mt
```

Comment:
Seems OK, but I'm not sure how it is meant to work? Can you point me at the corresponding cinder patch to handle this volume type?

Reply:
Thanks @yuval-lb . I'm fine with the doc update.